coverage/clover.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<coverage generated="1604901995640" clover="3.2.0">
+  <project timestamp="1604901995640" name="All files">
+    <metrics statements="0" coveredstatements="0" conditionals="0" coveredconditionals="0" methods="0" coveredmethods="0" elements="0" coveredelements="0" complexity="0" loc="0" ncloc="0" packages="0" files="0" classes="0"/>
+  </project>
+</coverage>
```

Comment:
Please add `coverage` to the .gitignore file, then follow this tutorial to remove these coverage files from the repository. For the time being, we don't really want these unnecessary files showing up in the Github repository:
https://www.codeblocq.com/2016/01/Untrack-files-already-added-to-git-repository-based-on-gitignore/